testutil/sample/launch.go

```diff
@@ -29,6 +29,7 @@ func Chain(id uint64, coordinatorID uint64) launch.Chain {
 		SourceHash:      String(10),
 		LaunchTriggered: false,
 		InitialGenesis:  launch.NewDefaultInitialGenesis(),
+		Metadata:        []byte(String(20)),
```

Comment:
```suggestion
		Metadata:        Bytes(20),
```
I also think we can create a `Metadata` sample method